<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPSNProxy</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qpsnres.qrc">
    <normaloff>:/main/resources/images/psv_icon.svg</normaloff>:/main/resources/images/psv_icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Console</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="consoleComboBox">
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PS Vita</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PSP/PSOne</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PS3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PS4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Other</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="plusCheckBox">
          <property name="text">
           <string>Show only PS+ content</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="statusBox">
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>New</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Downloading</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Completed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Paused</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="downloadTableWidget">
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column/>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="filterLabel">
          <property name="text">
           <string>Filter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="FilterLineEdit" name="downloadFilter">
          <property name="styleSheet">
           <string notr="true">FilterLineEdit { color:gray; font-style:italic; }</string>
          </property>
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resumeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Resume All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pauseButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Pause All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Setti&amp;ngs</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_package_directory"/>
    <addaction name="actionClear_downloaded_packages"/>
    <addaction name="actionClear_PSN_login_cookies"/>
    <addaction name="actionClear_thumbnail_cache"/>
    <addaction name="actionClear_download_list_cache"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport">
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionOpen_package_directory">
   <property name="text">
    <string>&amp;Open package directory</string>
   </property>
  </action>
  <action name="actionClear_downloaded_packages">
   <property name="text">
    <string>&amp;Clear downloaded packages</string>
   </property>
  </action>
  <action name="actionClear_PSN_login_cookies">
   <property name="text">
    <string>Clear &amp;PSN login cookies</string>
   </property>
  </action>
  <action name="actionClear_thumbnail_cache">
   <property name="text">
    <string>C&amp;lear thumbnail cache</string>
   </property>
  </action>
  <action name="actionClear_download_list_cache">
   <property name="text">
    <string>Clear &amp;download list cache</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Opt&amp;ions</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>filterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qpsnres.qrc"/>
 </resources>
 <connections/>
</ui>
